### PR TITLE
Fixed tiny bug in Migration Macros

### PR DESCRIPTION
**Pull request by fppt**

### CODE REVIEW

File: grakn-core/src/main/java/ai/grakn/graql/macro/Macro.java [hunk 1]
```diff
@@ -38,10 +38,6 @@
     @CheckReturnValue
     T apply(List<Object> values);
 
-    /**
-     * The name of the macro
-     * @return the name of the macro
-     */
     @CheckReturnValue
     String name();
```
Comment (aelred) [inline grakn-core/src/main/java/ai/grakn/graql/macro/Macro.java]: how will we ever know this is the "name" now???

**Comment by fppt** [comment]
!rtg

**Comment by fppt** [comment]
!rtg

**Comment by fppt** [comment]
!rtg

**Comment by fppt** [comment]
!rtg - ai.grakn.graql.internal.query.match.MatchModifierTest.ai.grakn.graql.internal.query.match.MatchModifierTest

Due to no host exception.

**Pull request merged**